backend/src/Repository/OrderLogsEntityRepository.php

```diff
@@ -19,32 +19,35 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, OrderLogsEntity::class);
     }
 
-    // /**
-    //  * @return OrderLogsEntity[] Returns an array of OrderLogsEntity objects
-    //  */
-    /*
-    public function findByExampleField($value)
+    public function getOrderLogsByOrderId(int $orderId): ?array
     {
-        return $this->createQueryBuilder('o')
-            ->andWhere('o.exampleField = :val')
-            ->setParameter('val', $value)
-            ->orderBy('o.id', 'ASC')
-            ->setMaxResults(10)
+        return $this->createQueryBuilder('orderLogsEntity')
+            ->select('orderLogsEntity.id, orderLogsEntity.createdAt, orderLogsEntity.orderState')
+
+            ->andWhere("orderLogsEntity.orderId = :orderId")
+            
+            ->setParameter('orderId',$orderId)
+            
+            ->orderBy('orderLogsEntity.id', 'DESC')
+
             ->getQuery()
-            ->getResult()
-        ;
+            ->getResult();
     }
-    */
 
-    /*
-    public function findOneBySomeField($value): ?OrderLogsEntity
+    public function getCurrentStageDate(int $orderId): ?array
```

Comment:
This function seems it returns the last status that the order take, but from its name it seems that it returns the last **status date** that the order take.